examples/bridge-app/linux/include/Device.h

```diff
@@ -61,7 +65,8 @@ class Device
 
 protected:
     bool mReachable;
-    char mName[kDeviceNameSize];
+    char mName[kDeviceNameSize + 1];
+    char mUniqueId[kDeviceUniqueIdSize + 1];
```

Comment:
Do we need to make sure that is it initialized with `\0`

Reply:
acked